lib/Dialect/ESI/ESITypes.cpp

```diff
@@ -21,6 +21,9 @@
 using namespace circt;
 using namespace circt::esi;
 
+/*
```

Comment:
You can delete this code. I assume it didn't compile with the new LLVM and switching to `assemblyFormat` was the simpler thing to do?

Reply:
I didn't even find the point where `ChannelPort` is defined - I assume it's now using some default `assemblyFormat`.